src/python/bot/fuzzers/libFuzzer/stats.py

```diff
@@ -262,6 +266,10 @@ def parse_performance_features(log_lines,
       stats['startup_crash_count'] = 0
       stats['edges_total'] = int(match.group(2))
 
+    match = LIBFUZZER_EXTRA_COUNTERS_REGEX.match(line)
```

Comment:
I am thinking if we even need this LIBFUZZER_EXTRA_COUNTERS_REGEX at all. How about just do 
stats['startup_crash_count'] = 0 as next line in libfuzzer_inited = True branch.

Reply:
Good idea, but `calculate_log_lines` doesn't touch the stats as of now. We can rely on `libfuzzer_lines_count` being non-zero, as that's essentially the same logic. Done, thanks for the suggestion!